### ISSUE TITLE
Cannot add items to a custom palette using keyboard

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#24520]
### Issue type

UI bug (incorrect info or interface appearance)

### Description with steps to reproduce

It seems the only way to add items to a custom palette currently is by drag and drop with a mouse.  Suggestions to fix this from the blind community are for an option to copy and paste or add from a context menu.

Or the following from Marc Sabatella:
The important thing in palette customization is that any element from the score can be added to any palette.  So this means you have to be able to select a score element and select a palette.  Since this is already basically how you add palette items to the score - select a target, navigate to a palette item, hit Enter to add the palette item to the select score element), it seems the same basic model should make sense for customization.  Select the score element you want to add to a palette, then navigate to a palette item, then hit some key *other* than Enter to add the selected score element into the current palette cell (shifting everything else later).  Could also be part of the context menu ("insert selected score element into palette").

This isn't necessarily better than the copy/paste idea.  But it does have the advantage of more closely mimicking how the palette works otherwise, and it does save a keystroke since no "copy" operation is needed.  Also preserves the contents of your clipboard.

### Supporting files, videos and screenshots

N/A

### What is the latest version of MuseScore Studio where this issue is present?

4.4.1

### Regression

I was unable to check

### Operating system

windows 11

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by cbjeukendrup** [comment]
Personally I think I would be in favour of the copy/paste solution since this would also make the feature more discoverable for users who don't necessarily depend on using the keyboard only. So you'd copy the item on the score view, then navigate to the palette, press a key to bring up a context menu, and choose "Paste". 

Another idea, to save a key press: make the Ctrl+V shortcut context-aware. So when you have focused the palettes panel, paste into the focused palette, otherwise paste onto the score. Or is that a bad idea?

**Comment by jmwmusic** [comment]
Yes, most people who commented on the accessibility group list suggested Ctrl C and Ctrl V for copy and paste.  I think these are some of the best known keyboard commands by any user and therefore the thing people will try automatically without having to look up how in a handbook.  I like the insert to palette in a context menu also. Having both options would be even better.

**Comment by jmwmusic** [comment]
This also needs to include design for how to remove both whole Palettes and remove Palette items, both presumably also currently done with mouse drag.  It is also essential to ensure that this process isn't something that can easily be done by mistake, so ideally from a context menu or a special palette customisation mode you enter to paste/delete etc.  The context menu both ways does seem most straightforward and safest to avoid accidental deletions or pasting to the wrong place.

**Comment by MarcSabatella** [comment]
You can remove palette items and entire palettes with the Delete key on your keyboard.
